blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/OrganizationFolderPipelineImpl.java

```diff
@@ -33,6 +32,11 @@ public OrganizationFolderPipelineImpl(OrganizationFolder folder, Link parent) {
 
     }
 
+    @Exported
+    public String getIcon() {
+        return folder.getIcon().getImageOf("32x32");
```

Comment:
@i386  I tried with cloudbeers org and it's giving correct icon url, https://avatars.githubusercontent.com/u/4181899?v=3&s=32.